spec/send_sms_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'send_sms'
+
+describe SendSMS do
+
+  subject(:sendsms) { described_class.new(order, total) }
+  let(:order) { double() }
```

Comment:
Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments. (https://github.com/bbatsov/ruby-style-guide#no-args-no-parens)